### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by John-Lin**

### CODE REVIEW

**Comment by chenyunchen** [comment]
Git

**Comment by hwchiu** [comment]
what is GT?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # 線上紅娘
 
-尋找人生下一春，就從 GIt 開始\(誤
+尋找人生下一春，就從 Gt 開始\(誤
```
Comment (kkdai) [inline README.md:3]: Why not Tesla?

Comment (John-Lin) [reply README.md:3]: 打錯字⋯

Comment (hwchiu) [reply README.md:3]: 還不快修XD

**Comment by John-Lin** [comment]
please check #8

**Pull request closed without merging**